packages/gestalt/src/Dropdown.js

```diff
@@ -183,7 +183,7 @@ export default function Dropdown({
       id={id}
       idealDirection={idealDirection}
       onDismiss={onDismiss}
-      positionRelativeToAnchor={false}
+      positionRelativeToAnchor={isWithinFixed}
```

Comment:
GitHub won't let me comment on the line directly for some reason, but there's a reference to dangerouslyRemoveLayer on line 201 that needs to be updated.

I'm also surprised your linter didn't catch that? You might want to double-check that it's working correctly

Reply:
Yeah, I had issues with the linter. I updated and ran the linter with no error. Fixed.